docs/en/ingest-management/release-notes/release-notes-8.5.asciidoc

```diff
@@ -22,6 +23,28 @@ Also see:
 * {kibana-ref}/release-notes.html[{kib} release notes]
 * {beats-ref}/release-notes.html[{beats} release notes]
 
+// begin 8.5.2 relnotes
+
+[[release-notes-8.5.2]]
+== {fleet} and {agent} 8.5.2
+
+Review important information about the {fleet} and {agent} 8.5.2 release.
+
+[discrete]
+[[bug-fixes-8.5.2]]
+=== Bug fixes
+
+{fleet}::
+* Fix known issue with adding {fleet-server} integration on Windows by always using posix paths for zip files {kib-issue}/144880[#144880] {kibana-pull}/144899[#144899]
```

Comment:
@juliaElastic I am going to merge this as soon as the build passes, but when you have a minute, can you just confirm that this is correct. There was no changelog in 144899, and it wasn't clear in the notes that that PR resolved 144880.

Reply:
LGTM, thanks! cc @hop-dev @kpollich